Ru translate database as a service (#3079)
cncf/glossary: content/ru/database-as-a-service.md

---
title: Database as a Service (DBaaS)
status: Deprecated
category: Technology
draft: true
tags: ["", "", ""]
---

> Database-as-a-Service (DBaaS, база данных как услуга) — это сервис, управляемый
> обачным провайдером (публичным или приватным) и позволяющий поддерживать приложения,
> не требуя от команды разработчиков традиционных функций администрирования баз данных.

Reviewer: ```suggestion
Database-as-a-Service (DBaaS, база данных как услуга) — сервис, предоставляемый [облачным](/ru/cloud-computing) провайдером (публичным или приватным). 
Позволяет запускать приложения, автоматизируя администрирование баз данных.
Благодаря DBaaS разработчики приложений могут использовать базы данных, не являясь в них экспертами и не нанимая администратора баз данных (DBA) для поддержания баз данных в актуальном состоянии.
```

## Проблема, которую он решает

> При локальном развертывании баз данных организациям регулярно приходится вкладывать средства в
> дополнительные хранилища и вычислительные мощности для их расширения, что может быть дорогостоящим процессом. 
> Кроме того, для предоставления и настройки базы данных требуются отдельные команды ИТ-инфраструктуры,
> что снижает скорость развертывания зависящих от них приложений.
> Их загрузка и выполнение занимают больше времени.

Reviewer: ```suggestion
При локальном развертывании баз данных организациям приходится регулярно вкладывать средства в
дополнительное место для хранения и вычислительные мощности для их расширения, что дорого обходится. 
Кроме того, для предоставления и настройки базы данных требуются отдельные команды ИТ-инфраструктуры,
что снижает скорость развертывания приложений на их основе.
Загрузка БД и работа с ними также занимает больше времени.
```

## Как помогает DBaaS

> DBaaS позволяет разработчикам передавать все административные операции поставщику облачных услуг на аутсорсинг. 
> Поставщик услуг обеспечивает бесперебойную работу базы данных,
> включая управление конфигурацией, резервное копирование, исправления, обновления, мониторинг служб и многое другое,
> предоставляя удобный интерфейс для управления и мониторинга. 
> DBaaS помогает организациям быстрее разрабатывать приложения корпоративного уровня при минимизации затрат на базу данных.

Reviewer: ```suggestion
DBaaS перекладывает всю ответственность за администрирование БД с разработчиков на поставщика облачных услуг. 
Поставщик услуг обеспечивает бесперебойную работу базы данных,
управляет ее конфигурацией, проводит резервное копирование, вносит исправления, устанавливает обновления, следит за сервисами и т. д.,
а пользователь в свое распоряжение получает удобный интерфейс для управления. 
DBaaS помогает организациям быстрее разрабатывать приложения корпоративного уровня, минимизируя затраты на базы данных.
```
